Intersect (Core)/Config/Options.cs

```diff
@@ -113,6 +113,8 @@ public class Options
         public static int RequestTimeout => Instance.PlayerOpts.RequestTimeout;
 
         public static int TradeRange => Instance.PlayerOpts.TradeRange;
+        
+        public static int MaxHotbar => Instance.PlayerOpts.MaxHotbar;
```

Comment:
We're trying to do away with top level options, can you please remove this and refer to the option through its lower level version?

Reply:
👍🏽 Commited.